olp-cpp-sdk-dataservice-read/src/CatalogClient.cpp

```diff
@@ -17,19 +17,19 @@
  * License-Filename: LICENSE
  */
 
-#include "olp/dataservice/read/CatalogClient.h"
+#include <olp/dataservice/read/CatalogClient.h>
```

Comment:
For own library includes you dont need to use <>.
Only for different libs, e.g. core.